Makefile

```diff
@@ -380,7 +370,7 @@ create-cluster: $(CLUSTERCTL) $(ENVSUBST) ## Create a development Kubernetes clu
 
 	# Deploy calico
 	kubectl --kubeconfig=./kubeconfig apply -f https://docs.projectcalico.org/manifests/calico.yaml
-	# FIXME:
+
 	# Create a worker node with MachineDeployment.
 	kubectl apply -f examples/_out/machinedeployment.yaml
 
```

Comment:
Please update v0.3.0-rc.3 to v0.3.0 in line 340.

Reply:
Done